src/main/java/seedu/address/model/person/Email.java

```diff
@@ -9,13 +9,15 @@
  */
 public class Email {
 
+    private static final int MAXIMUM_LENGTH = 100;
```

Comment:
The longest active email address seems to be 345 characters. To be inclusive, perhaps the limit could be higher? Maybe 400?